--- source/syn/board/ia840f/syn_top/ofs_pr_afu.qsf
+++ work02/syn/board/ia840f/syn_top/ofs_pr_afu.qsf
@@ -24,7 +24,6 @@
 ############################################################################################
 set_global_assignment -name TOP_LEVEL_ENTITY top
 set_global_assignment -name PROJECT_OUTPUT_DIRECTORY output_files
-set_global_assignment -name OPTIMIZATION_MODE "SUPERIOR PERFORMANCE"
 set_global_assignment -name OPTIMIZATION_TECHNIQUE SPEED
 set_global_assignment -name OPTIMIZE_POWER_DURING_SYNTHESIS OFF
 set_global_assignment -name OPTIMIZE_POWER_DURING_FITTING OFF
@@ -37,7 +36,7 @@
 set_global_assignment -name REMOVE_DUPLICATE_REGISTERS OFF -to rst_n_sys_mem
 set_global_assignment -name REMOVE_DUPLICATE_REGISTERS OFF -to rst_n_sys_pcie
 
-set_global_assignment -name LAST_QUARTUS_VERSION "22.3.0 SP0.31,0.35 Pro Edition"
+set_global_assignment -name LAST_QUARTUS_VERSION "26.1.1 Pro Edition"
 
 set_global_assignment -name FLOW_DISABLE_ASSEMBLER OFF
 set_global_assignment -name FLOW_ENABLE_INTERACTIVE_TIMING_ANALYZER OFF
@@ -141,3 +140,4 @@
 # VAB Assignments 
 ############################################################################################
 set_global_assignment -name ENABLE_MULTI_AUTHORITY ON
+set_global_assignment -name OPTIMIZATION_MODE "HIGH PERFORMANCE EFFORT" -comment "Use High Performance Effort optimization mode when optimizing for performance. Superior Performance optimization mode has been deprecated and will be removed in 27.1.1."

--- source/syn/board/ia840f/syn_top/ofs_top.qpf
+++ work02/syn/board/ia840f/syn_top/ofs_top.qpf
@@ -1,8 +1,32 @@
-# Copyright (C) 2020 Intel Corporation.
-# SPDX-License-Identifier: MIT
+# -------------------------------------------------------------------------- #
+#
+# Copyright (C) 2026  Altera Corporation. All rights reserved.
+# Your use of Altera Corporation's design tools, logic functions 
+# and other software and tools, and any partner logic 
+# functions, and any output files from any of the foregoing 
+# (including device programming or simulation files), and any 
+# associated documentation or information are expressly subject 
+# to the terms and conditions of the Altera Program License 
+# Subscription Agreement, the Altera Quartus Prime License Agreement,
+# the Altera IP License Agreement, or other applicable license
+# agreement, including, without limitation, that your use is for
+# the sole purpose of programming logic devices manufactured by
+# Altera and sold by Altera or its authorized distributors.  Please
+# refer to the Altera Software License Subscription Agreements 
+# on the Quartus Prime software download page.
+#
+# -------------------------------------------------------------------------- #
+#
+# Quartus Prime
+# Version 26.1.1 Build 130 08/06/2026 SC Pro Edition
+# Date created = 01:20:37  September 18, 2026
+#
+# -------------------------------------------------------------------------- #
 
-#
+QUARTUS_VERSION = "26.1"
+DATE = "01:20:37  September 18, 2026"
+
 # Revisions
 
 PROJECT_REVISION = "ofs_top"
-
+PROJECT_REVISION = "ofs_pr_afu"

--- source/syn/board/ia840f/syn_top/ofs_top.qsf
+++ work02/syn/board/ia840f/syn_top/ofs_top.qsf
@@ -24,7 +24,6 @@
 ############################################################################################
 set_global_assignment -name TOP_LEVEL_ENTITY top
 set_global_assignment -name PROJECT_OUTPUT_DIRECTORY output_files
-set_global_assignment -name OPTIMIZATION_MODE "SUPERIOR PERFORMANCE"
 set_global_assignment -name OPTIMIZATION_TECHNIQUE SPEED
 set_global_assignment -name OPTIMIZE_POWER_DURING_SYNTHESIS OFF
 set_global_assignment -name OPTIMIZE_POWER_DURING_FITTING OFF
@@ -37,7 +36,7 @@
 set_global_assignment -name REMOVE_DUPLICATE_REGISTERS OFF -to rst_n_sys_mem
 set_global_assignment -name REMOVE_DUPLICATE_REGISTERS OFF -to rst_n_sys_pcie
 
-set_global_assignment -name LAST_QUARTUS_VERSION "22.3.0 SP0.31,0.35 Pro Edition"
+set_global_assignment -name LAST_QUARTUS_VERSION "26.1.1 Pro Edition"
 
 set_global_assignment -name FLOW_DISABLE_ASSEMBLER OFF
 set_global_assignment -name FLOW_ENABLE_INTERACTIVE_TIMING_ANALYZER OFF
@@ -126,3 +125,4 @@
 # VAB Assignments 
 ############################################################################################
 set_global_assignment -name ENABLE_MULTI_AUTHORITY ON
+set_global_assignment -name OPTIMIZATION_MODE "HIGH PERFORMANCE EFFORT" -comment "Use High Performance Effort optimization mode when optimizing for performance. Superior Performance optimization mode has been deprecated and will be removed in 27.1.1."
